data-prepper-plugins/opensearch/README.md

```diff
@@ -209,6 +209,7 @@ With the `document_root_key` set to `status`. The document structure would be `{
     duration: "15 ms"
 }
 ```
+- `backend_version`: A String indicating whether the sink backend version is Elasticsearch 6 or above (i.e. Elasticsearch 7.x or OpenSearch). `es_6` represents Elasticsearch 6; `null` represents Elasticsearch 7.x or OpenSearch. Default to `null`.
```

Comment:
Maybe name this `distribution_version`? That might be closer to the terms used in OpenSearch itself which uses "distribution" and "version".